ntp-udp/src/socket.rs

```diff
@@ -304,13 +305,14 @@ fn recv(
 
             #[cfg(target_os = "linux")]
             ControlMessage::ReceiveError(_error) => {
-                warn!("unexpected error message on the MSG_ERRQUEUE");
+                tracing::info!("unexpected error message on the MSG_ERRQUEUE");
             }
 
             ControlMessage::Other(msg) => {
-                warn!(
+                tracing::info!(
                     "weird control message {:?} {:?}",
-                    msg.cmsg_level, msg.cmsg_type
+                    msg.cmsg_level,
+                    msg.cmsg_type
```

Comment:
This feels like it is either a bug (and should thus be warn) or some os oddity that we just want to log as debug.